css/style.scss

```diff
@@ -0,0 +1,359 @@
+}
+#header a{
+    display: inline-block;
+}
+#header ul {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    float: right;
+}
+#header ul li {
+    float: left;
+    font-family: 'Montserrat', sans-serif;
+    font-size: 13px;
+    font-weight: 600;
+}
+#header ul li:hover {
+    border-radius: 4px;
+}
+#header ul li a{
+    text-decoration: none;
+    color: $dark-brown;
+    padding: 10px 15px;
+    text-transform: uppercase;
+}
+#header ul li a:active{
+    color: $cgreen-color; 
+}
+#header ul li a:hover{
+    color: $cgreen-color; 
+}
+#navigation {
+    position: relative;
+    float: right;
+  }
+  
+  #navigation > .navigation-header {
+    display: inline;
+  }
+  #navigation > .navigation-header > .navigation-title {
+    display: inline-block;
+    font-size: 22px;
+    color: #fff;
+    padding: 10px 10px 10px 10px;
+  }
+  #navigation > .navigation-btn {
+    display: none;
+  }
+  #navigation > .navigation-links {
+    display: flex;
+    float: right;
+    font-size: 13px;
+    font-weight: 400;
+    flex-wrap: nowrap;
+  }
+  #navigation > .navigation-links > a {
+    display: inline-block;
+    padding: 13px 10px 13px 10px;
+    text-decoration: none;
+    color: #efefef;
+  } 
+  #navigation > #navigation-check {
+    display: none;
+  }
+  #wrap{
+    background-color: #ddd;
+  }
+  #slider{
+    width: 100%;
+    margin: 0 auto;
+    list-style: none;
+    padding: 93px 0 0 0;
+
+  }
+  .slide-item{
+    img{
+      width: 100%;
+    }  
+  }
+  .banner-slide{
+    position: relative;
+}
+.banner-caption {
+  max-width: 50%;
+}
+.banner-img {
+  width: 50%;
+}
+.banner-img img {
+  width: 100%;
+}
+.carousel-caption h1 {
+  font-size: 60px;
+  font-weight: 400;
+  margin-top: 118px;
+  margin-bottom: 24px;
+  line-height: 70px;
+}
+.banner-caption p {
+  font-weight: 300;
+  line-height: 1.5;
+  width: 80%;
+  font-size: 16px;
+  color: #9a9a9a;
+}
+.carousel-caption {
+  position: absolute;
+  z-index: 10;
+  color: #fff;
+  max-width: 1100px;
+  margin: 0 auto;
+  left: 0;
+  right: 0;
+  top: 24px;
+  display: flex;
+  flex-wrap: wrap;
+}
+.gap{
+    padding: $section-gap;
+}
+.about{
+    text-align: center;
+}
+h2{
+    font-size: 30px;
+}
+.about p{
+    font-size: 16px;
+    line-height: 24px;
+    max-width: 670px;
+    margin: 0 auto;
+}
+.about span{
+    display: inline-block;
+    margin-top: 40px;
+}
+.bg-color{
+    background-color: $cgreen-color;
+}
+.clients ul{
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+}
+.clients ul li{
+    flex: 0 0 25%;
+    display: flex;
+}
+.project{
+    position: relative;
+}
+.project-text {
+    margin: 0 0 0 20px;
+}
+.project-text h5 {
+    margin: 0;
+    font-size: 18px;
+    color:$white-color;
+}
+.project-text p {
+    margin: 6px 0 0 0;
+    font-size: 12px;
+    color: #fff;
+    text-transform: uppercase;
+}
+// #gallery{
+//     display: flex;
+// }
+#gallery ul{
```

Comment:
I'm unable to fine any id containing `gallery`.

Reply:
i have not seen this id in css and Sass file in 3rd task file

Reply:
Then I think we should remove the redundant css from the file.

Reply:
it's second task css

Reply:
Is our second task related to the 3rd one ?